documentation/src/physical_types/spins.md

```diff
@@ -168,27 +168,6 @@ hamiltonian.add_operator_product(pp, 1.0.into()).unwrap();
 println!("{}", hamiltonian);
 ```
 
-## Operators and Hamiltonians
-
-### Examples
-
-```rust
-use qoqo_calculator::CalculatorComplex;
-use struqture::prelude::*;
-use struqture::spins::{PauliProduct, QubitOperator};
-
-let mut operator = QubitOperator::new();
-
-let pp = PauliProduct::new().x(0).z(2);
-operator
-    .add_operator_product(pp, CalculatorComplex::new(1.0, 1.5))
-    .unwrap();
-operator
-    .add_operator_product(PauliProduct::new().z(3), CalculatorComplex::new(1.0, 1.5))
-    .unwrap();
-println!("{}", operator);
-```
-
 The equivalent code in python:
```

Comment:
This line doesn't make much sense with the rust example removed

Reply:
The Rust example (and the title) was duplicated, if you open the lines a little further up, you'll see that it's present there too

Reply:
Yes but the rust example is more complete that the python one so they're not really equivalent, I think we should add the operator.set and hamiltonian part to the python example

Reply:
Ah good point! I'll add this.

Reply:
I have updated the python code snippet